deploy.sh

```diff
@@ -140,13 +140,13 @@ function retry_terraform {
 
 function setup_terraform {
   TF_LATEST=$(curl -s https://api.github.com/repos/hashicorp/terraform/releases/latest | grep tag_name | cut -d'"' -f4)
-  if which terraform > /dev/null 2>&1; then
-    TF=$(which terraform 2> /dev/null)
-  fi
+  EXT_PATH=$(which terraform 2> /dev/null)
```

Comment:
This results in a non-zero error code when terraform is not present in the path, which causes the script to fail.

Possible Solution - `EXT_PATH=$(which terraform 2> /dev/null || true)`